src/contentmanager.cpp

```diff
@@ -53,7 +53,7 @@ ContentManager::ContentManager(Library* library, kiwix::Downloader* downloader,
     // mp_view will be passed to the tab who will take ownership,
     // so, we don't need to delete it.
     mp_view = new ContentManagerView();
-    managerModel = new ContentManagerModel(this);
```

Comment:
No, the commit message is correct. At least it reflects my understanding of `ContentManager`'s and `ContentManagerModel`'s roles in the current design:

- `ContentManager` manages full content
-  `ContentManagerModel` represents only the currently displayed (i.e. filtered rather than visible) content